aws/aws-encryption-sdk-dafny
# chore(Test Vector): Add interop test for Go

*Issue #, if available:*

*Description of changes:*
- Adds/Updates makefile, externs and go.mod file to run test vectors.

*How to test this?*
- There are two ways to test it. Use either (i) or (ii).

    1. Run `make test_go`
    2. Run following commands:
         -  make `test_generate_vectors_go`
         - make `test_encrypt_vectors_go`
         - make `test_decrypt_encrypt_vectors_go`

*Squash/merge commit message, if applicable:*
```
chore(Test Vector): Add interop test for Go
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
